+--- gimp-GIMP_3_0_0_RC3/desktop/gimp.desktop.in.in.old  2025-02-12 20:11:39.862568018 -0500
+++ gimp-GIMP_3_0_0_RC3/desktop/gimp.desktop.in.in      2025-02-12 20:12:42.588692364 -0500
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Name=GNU Image Manipulation Program
+# SUN_BRANDING
+Name=Image Editor
 GenericName=Image Editor
-Comment=Create images and edit photographs
+# SUN_BRANDING
+Comment=Create, edit or convert images between standard graphics formats
 # TRANSLATORS: Search terms to find this application.
 # 1. Do NOT translate or localize the semicolons!
 # 2. The list MUST also end with a semicolon!
